definitions/variable/economy/economy.yaml

```diff
@@ -2,69 +2,69 @@
 
 - Consumption:
     description: Total consumption of all goods, by all consumers in a region
-    unit: [million EUR_2020/yr, billion USD_2010/yr]
+    unit: [million EUR_2020/yr, billion USD_2010/yr, billion EUR_2020/yr]
```

Comment:
Let us please use only either million EUR or billion EUR - having the same unit at different orders of magnitude is basically an invitation for confusion...

Reply:
totally fine with this - I was afraid that if I remove something, I would interfere with the use of the nomenclature in another project. 
I take your comment as implying that I can remove the million EUR unit, so I will do a commit with this